docs/administration/authentication/active-directory-authentication/index.md

```diff
@@ -44,6 +44,18 @@ Octopus is built on top of HTTP.sys, the same kernel driver that IIS is built on
 When the link is clicked, it redirects to a page which is configured to tell HTTP.sys to issue the browser challenge. The browser and HTTP.sys negotiate the authentication just like an IIS website would. The user principal is then passed to Octopus. Octopus will then query Active Directory for other information about the user.
 :::
 
+### Using Kerberos {#ActiveDirectoryauthentication-UsingKerberos}
+
+By default `IntegratedWindowsAuthentication` will try to use Kerberos first, and then fall back to `NTLM` if it fails.
```

Comment:
Should it say `Negotiate` instead of `Kerberos` ?

Reply:
Perhaps, I'm actually still unclear what the different between negotiate and Integrated windows auth is to be honest. To me, it seems that `Negotiate` and `IntegratedWindowsAuthentication` are the same?

From the above MS link:
```
The Negotiate security header lets clients select between Kerberos authentication and NTLM authentication. 
```

I wanted to talk about kerberos specifically as that is what sysadmins will be thinking in terms of.